### PR TITLE
Made fixes in layout and alignment

### PR DESCRIPTION
**Pull request by sweta1308**
Made alignment and layouting fixes.

### CODE REVIEW

File: src/pages/development.tsx [hunk 1]
```diff
@@ -69,29 +69,31 @@ const Development = () => {
...
+              <span className="mx-2 lg:text-[24px] font-normal leading-[32px]">
+                Try OpenMetadata as a managed service for free, from Collate.
+              </span>
+              <span className="lg:relative lg:top-[6px]">
+                <Link
+                  className="border-2 border-[#D3ECF7] px-3 py-2 lg:text-[32px] leading-[25px] hover:text-[#D3ECF7]"
+                  href="https://getcollate.io"
+                  target="_blank"
+                >
+                  Get Started
```
Comment (Sachin-chaurasiya) [inline src/pages/development.tsx:93]: Let's create a separate component for this strip.

Comment (sweta1308) [reply src/pages/development.tsx]: Updated

File: src/components/Achievement/Achievement.tsx [hunk 2]
```diff
@@ -3,8 +3,8 @@ import Image from "next/image";
 
 const Achievement = () => {
   return (
-    <div className="relative max-w-[1440px] mx-auto">
-      <div className="relative z-[11] py-16 grid gap-8 sm:px-0 md:px-16 md:py-18 lg:grid-cols-2 lg:gap-0">
+    <div className="bg-gradient-to-b from-[#F1EDFD] to-white">
+      <div className="max-w-[1440px] mx-auto py-16 grid gap-8 sm:px-0 md:px-16 md:py-18 lg:grid-cols-2 lg:gap-0">
```
Comment (Sachin-chaurasiya) [inline src/components/Achievement/Achievement.tsx:7]: @sweta1308, let's create a class and put all of this repetitive styling in there.

```css
.custom-container{
@apply max-w-[1440px] mx-auto;
}
```

Comment (sweta1308) [reply src/components/Achievement/Achievement.tsx]: Updated

**Review by Sachin-chaurasiya — APPROVED** [review]

**Pull request merged**